File: 1010.py
p1 = input()
quantity1 = int(input())
price1 = float(input())
p2 = input()
quantity2 = int(input())
price2 = float(input())
pr1 = quantity1 * price1 
pr2 = quantity2 * price2
total = pr1 + pr2
print("VALOR A PAGAR: R$ " + "{:.2f}".format(total))

File: 1010.c
#include<stdio.h>
int main(){
int n1,q1;
float p1,total,total1,total2;
scanf("%d%d%f",&n1,&q1,&p1);
total=(q1 * p1);
scanf("%d%d%f",&n1,&q1,&p1);
total +=(q1 * p1);
printf("VALOR A PAGAR: R$ %.2f",total);

return 0;
}
